doc/user/content/sql/create-table.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "CREATE TABLE"
+description: "`CREATE TABLE` creates a non-streaming, in-memory data source."
+menu:
+  main:
+    parent: 'sql'
+---
+
+`CREATE TABLE` creates a non-streaming, in-memory data source.
+
+## Conceptual framework
+
+Tables store non-streaming data that is inserted via [INSERT](../insert) statements.
+
+### When to create a table
+
+You might want to create a table when...
+
+- You want to manually insert rows of data into Materialize.
+- You want to test Materialize's features without setting up a data stream.
+
+## Syntax
+
+{{< diagram "create-table.svg" >}}
+
+Field | Use
+------|-----
+_table&lowbar;name_ | A name for the table.
+_col&lowbar;name_ | The name of the column to be created in the table.
+_col&lowbar;type_ | The data type of the column indicated by _col_name_.
+_NOT NULL_ | Provide a `NOT NULL` constraint if a column is not nullable. All columns without this constraint will be nullable.
+
+## Details
+
+### Restrictions
+
+Tables will not persist any data that is inserted. This means that restarting a
```

Comment:
Can you change this from future to present tense? i.e. "Tables do not persist..." etc.